polyply/data/martini3/dextran.martini3.ff

```diff
@@ -4,9 +4,9 @@
 
 [ atoms ]
 ; nr type resnr residue    atom cgnr charge mass
-  1  SP3   1   DEX    A    1    0
-  2  SP3   1   DEX    B    2    0
-  3  SP2   1   DEX    C    3    0
+  1  SP3   1   DEX    A    1    0  54.0
+  2  SP3   1   DEX    B    2    0  54.0
+  3  SP2   1   DEX    C    3    0  54.0
```

Comment:
nitpick alert (aka, no real issue and feel free to ignore): maybe write masses either as 54 or 54.0 consistently across Martini files

Reply:
at some point I'll probably have to rewrite the polymer library for M3 entirely considering we have more parameters in the pipline. For now this is not such a big deal because it will be printed as floats always. This is the vermouth format definition.